decidim-core/app/controllers/decidim/searches_controller.rb

```diff
@@ -13,7 +13,7 @@ class SearchesController < Decidim::ApplicationController
     def index
       Search.call(term, current_organization, filters) do
         on(:ok) do |results|
-          expose(results: results)
+          expose(results: results.page(params[:page]).per(params[:per_page]))
```

Comment:
Replace with `@results = results.page(params[:page]).per(params[:per_page])`. The `expose` method is only useful for when we use `rectify`'s presenters, and we don't

Reply:
I had to use this method so that specs can "see" this attribute.

Reply:
Just to try it, can you add `@results = nil` in between lines 13-14, and then `@results = results.page(params[:page]).per(params[:per_page])` inside the block (instead of the `expose`)?

If this works, then I'd rather use the `expose` method!

Reply:
Your proposal is dirty code. What is exactly the problem with `expose`?

Reply:
Reading the README for Rectify I even think that using `expose` is the correct way to expose values to the view:
> You may occasionally want to expose a value within a handler block to the view. You do this via the expose method within the handler block. If you want to use expose then you must include the Rectify::ControllerHelpers module in your controller. You pass a hash of the variables you wish to expose to the view and they will then be available. If you have set a Presenter for the view then expose will try to set an attribute on that presenter. If there is no Presenter or the Presenter doesn't have a matching attribute then expose will set an instance variable of the same name. See below for more details about Presenters.

Reply:
@tramuntanal I'm aware my proposal is noisy code. That's why I say that, if my proposal works, use `expose`. I say this to be sure of what does `expose` do exactly.

TLDR: use `expose` 👍

Reply:
Sorry from my side also, I read your "I'd rather use the expose method!" but thought it laked a "no" by error. :sweat_smile: